app/src/main/java/org/mozilla/focus/fragment/BrowserFragment.java

```diff
@@ -285,13 +292,24 @@ public void onValueChanged(@NonNull Boolean loading) {
 
         blockView = (FrameLayout) view.findViewById(R.id.block);
 
-        lockView = (ImageView) view.findViewById(R.id.lock);
+        lockView = view.findViewById(R.id.security_info);
         session.getSecure().observe(this, new Observer<Boolean>() {
             @Override
             public void onChanged(Boolean secure) {
-                lockView.setVisibility(secure ? View.VISIBLE : View.GONE);
+                if (secure) {
+                    lockView.setImageResource(R.drawable.ic_lock);
+                } else {
+                    if (URLUtil.isHttpUrl(getUrl())) {
+                        // HTTP site
+                        lockView.setImageResource(R.drawable.ic_internet);
+                    } else {
+                        // Certificate is bad
+                        lockView.setImageResource(R.drawable.ic_warning_16);
```

Comment:
Let's remove the "_16" suffix from the image files. For a vector drawable that doesn't mean much.